account_invoice_production_lot/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+* Create and validate a sale order
+* Set *Lots/Serial Numbers* on the delivered lines by clicking on the button *Lot Split*
+* After creating the invoice,  the *Lots/Serial Numbers* are displayed in the *Production
+  Lots* on the invoice line form and in *formatted note* field on
+  the invoice report, and, in case **Tracking** has been selected to **By lots**, their corresponding delivered quantities will be displad as well
```

Comment:
Delivered quantities always are shown, in both cases, when we tracking product by lots and by and by unique serial number.

Reply:
I have adapted the code to comply with the README.